scripts/osfstorage/usage_audit.py

```diff
@@ -0,0 +1,99 @@
+"""File: usage_audit.py
+Find all users and projects where their total usage (current file + deleted files) is >= the set limit
+Projects or users can have their GUID whitelisted via `usage_audit whitelist [GUID ...]`
+User usage is defined as the total usage of all projects they have > READ access on
```

Comment:
I think this is the case, but I wanted to be sure - if a GUID is whitelisted, its usage is excluded from aggregate calculation, correct?

For instance:  The user limit was 100MB. A user is an admin on a project with 90MB of usage. The user is also a read/write contributor on a project with 120MB of usage, but this project's GUID is whitelisted. In this case, the user's usage is 90MB, correct?